### PR TITLE
Bloody Eye (Again, I'm sorry)

### PR DESCRIPTION
**Pull request by BetaCentari**
I'm so sorry Bokkie.

## About The Pull Request
Adds bloody eye hypospray (hopefully)

## Why It's Good For The Game
Does a cool last stand thing for traitors and gives them a last resort option to go into a rage.

## Changelog
:cl:
add: Bloody Eye Hypospray
imageadd: Sprites for hypospray
/:cl:

### CODE REVIEW

File: tgui/yarn.lock [hunk 1]
```diff
@@ -6037,12 +6037,12 @@ __metadata:
   languageName: node
   linkType: hard
 
-"marked@npm:^2.0.3":
-  version: 2.0.3
-  resolution: "marked@npm:2.0.3"
+"marked@npm:^4.0.10":
+  version: 4.0.10
+  resolution: "marked@npm:4.0.10"
```
Comment (Bokkiewokkie) [inline tgui/yarn.lock:6042]: This is actually not a good can you undo this change?

Comment (BetaCentari) [reply tgui/yarn.lock]: All changes in Yarnlock have been reverted.

File: tgui/yarn.lock [hunk 2]
```diff
@@ -6041,7 +6041,7 @@ __metadata:
   version: 2.0.3
   resolution: "marked@npm:2.0.3"
   bin:
-    marked: bin/marked
+    marked: bin/marked.js
```
[the remark this answers is not in the document]
Comment (BetaCentari) [reply tgui/yarn.lock]: Reverted.

File: nsv13/code/game/objects/items/bloody_eye.dm [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+			step(M, pick(GLOB.cardinals))
+	if(prob(20))
+		INVOKE_ASYNC(M, /mob.proc/emote, pick("scream", "laugh"))
+	if(prob(33))
+		M.visible_message("<span class='danger'>[M]'s eyes are blood red!</span>")
+		M.drop_all_held_items()
+	..()
+	M.adjustToxLoss(0.25, 0)
+	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, pick(0.1, 0.15, 0.2, 0.25))
+	. = 1
```
Comment (Archanial) [inline nsv13/code/game/objects/items/bloody_eye.dm:69]:
```suggestion
	return TRUE
```

File: nsv13/code/game/objects/items/bloody_eye.dm [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+	if((M.mobility_flags & MOBILITY_MOVE) && !ismovableatom(M.loc))
+		for(var/i = 0, i < 6, i++)
+			step(M, pick(GLOB.cardinals))
+	M.Jitter(150)
+	M.Dizzy(15)
+	M.adjustToxLoss(0.25, 0)
+	if(prob(50))
+		INVOKE_ASYNC(M, /mob.proc/emote, pick("scream", "laugh"))
+	..()
+	. = 1
```
Comment (Archanial) [inline nsv13/code/game/objects/items/bloody_eye.dm:104]:
```suggestion
	return TRUE
```

File: nsv13/code/game/objects/items/bloody_eye.dm [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+	status_type = STATUS_EFFECT_UNIQUE
+	tick_interval = 1
+	examine_text = "<span class='warning'>Their eyes are blood red!</span>"
+	alert_type = null
+
+/datum/status_effect/bloody_eye/tick()
+	owner.a_intent_change(INTENT_HARM)
+	if(owner.a_intent != INTENT_HARM)
+		owner.a_intent_change(INTENT_HARM)
+		to_chat(owner, "<span class='warning'>KILLKILLKILL</span>")
```
Comment (Archanial) [inline nsv13/code/game/objects/items/bloody_eye.dm:158]: doesn't kill

File: nsv13/code/game/objects/items/bloody_eye.dm [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+	M.AdjustKnockdown(-40, FALSE)
+	M.AdjustUnconscious(-40, FALSE)
+	M.AdjustParalyzed(-40, FALSE)
+	M.AdjustImmobilized(-40, FALSE)
+	M.adjustStaminaLoss(-30, 0)
+	M.Jitter(2)
+	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 0.25)
+	if(prob(5))
+		INVOKE_ASYNC(M, /mob.proc/emote, pick("scream", "laugh"))
+	..()
```
[the remark this answers is not in the document]
Comment (BetaCentari) [reply nsv13/code/game/objects/items/bloody_eye.dm:50]: I don't know what you mean by this.

File: nsv13/code/game/objects/items/bloody_eye.dm (hunk 5 above)
Comment (BetaCentari) [reply nsv13/code/game/objects/items/bloody_eye.dm:158]: I don't know what you mean by this but the point is that the drug puts a player into a bloodthirsty rage. And so if they try to change intent to something else, it'll send that message telling them to kill.

File: nsv13/code/game/objects/items/bloody_eye.dm (hunk 6 above)
[the remark this answers is not in the document]
Comment (Archanial) [reply nsv13/code/game/objects/items/bloody_eye.dm:50]: I mean calling parent function the ..()

File: nsv13/code/game/objects/items/bloody_eye.dm (hunk 5 above)
Comment (Archanial) [reply nsv13/code/game/objects/items/bloody_eye.dm:158]: It's because github cut my comment.
You are setting the intent to harm, you don't have to check if it's harm

Comment (BetaCentari) [reply nsv13/code/game/objects/items/bloody_eye.dm:158]: Gotcha.

File: nsv13/code/game/objects/items/bloody_eye.dm (hunk 6 above)
[the remark this answers is not in the document]
Comment (BetaCentari) [reply nsv13/code/game/objects/items/bloody_eye.dm:50]: Got it.

**Review by Bokkiewokkie — APPROVED** [review]
